modules/nodes-custom-scheduler-deploying.adoc

```diff
@@ -20,6 +20,34 @@ Information on how to create a scheduler binary is outside the scope of this doc
 
 .Procedure
 
+. Create a file that contains a config map that holds the scheduler configuration file:
++
+.Example `scheduler-config-map.yaml`
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: scheduler-config
+  namespace: kube-system <1>
+data:
+  scheduler-config.yaml: | <2>
+    apiVersion: kubescheduler.config.k8s.io/v1beta2
+    kind: KubeSchedulerConfiguration
+    profiles:
+      - schedulerName: custom-scheduler <3>
+----
+<1> This procedure uses the `kube-system` namespace, but you can use the namespace of your choosing.
+<2> When you define your `Deployment` resource later in this procedure, this file is passed in to the scheduler command by using the `--config` argument.
+<3> Define a scheduler profile for your custom scheduler. This is used when defining the `schedulerName` in the `Pod` configuration.
```

Comment:
The second sentence has a reference error where "This" needs a noun. Maybe "This scheduler profile"